xesmf/util.py

```diff
@@ -109,7 +109,7 @@ def _flatten_poly_list(polys):
     """Iterator flattening MultiPolygons."""
     for i, poly in enumerate(polys):
         if isinstance(poly, MultiPolygon):
-            for sub_poly in poly:
+            for sub_poly in poly.geoms:
```

Comment:
relation to current PR?

Reply:
I think this solves a DeprecationWarning.